index.html

```diff
@@ -3,12 +3,12 @@
 <html>
 	<head>
 		<meta charset = "utf-8">
-		<title>ideabox project </title>
-		<link href = "style/reset.css" rel="stylesheet" type="text/css">
-		<link href ="style/style.css" rel="stylesheet" type ="text/css">
+		<title>2doBox</title>
+		<link href="style/reset.css" rel="stylesheet" type="text/css">
+		<link href="style/style.css" rel="stylesheet" type="text/css">
 		<link href="https://fonts.googleapis.com/css?family=Roboto+Slab:700" rel="stylesheet">
 		<link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
-		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<meta name="viewport" content="width=device-width" initial-scale=1">
```

Comment:
I think the comma was correct initially. 
<meta name="viewport" content="width=device-width, initial-scale=1">
https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag